.github/workflows/test.yml

```diff
@@ -146,78 +195,48 @@ jobs:
-                    .
+        $PATH = [Environment]::GetEnvironmentVariable("PATH")
+        $graphviz_path = "C:\Program Files\Graphviz\bin"
+        [Environment]::SetEnvironmentVariable("PATH", "$graphviz_path;$PATH")
+        dot -c
+        dot -v
+        neato -V
+        Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+        python -m venv pgvenv
+        pgvenv\Scripts\Activate.ps1
+        python -m pip install --upgrade pip wheel setuptools
+        python -m pip install -r requirements/test.txt
+        python setup.py build_ext --include-dirs="C:\Program Files\Graphviz\include" --library-dirs="C:\Program Files\Graphviz\lib"
+        python setup.py install
         pip list
 
     - name: Test pygraphviz
       run: |
-        export PATH=/opt/local/bin:/opt/local/sbin:$PATH
-        source ~/venv/bin/activate
-        pytest --doctest-modules --durations=10 --pyargs pygraphviz
-
-
-#  windows:
-#
-#    runs-on: windows-latest
-#
-#    strategy:
-#      matrix:
-#        python-version: [3.7]
-#
-#    steps:
-#
-#    - name: Checkout pygraphviz
-#      uses: actions/checkout@v2
-#
-#    - name: Set up Python ${{ matrix.python-version }}
-#      uses: actions/setup-python@v2
-#      with:
-#        python-version: ${{ matrix.python-version }}
-#
-#    - name: Download graphviz source
-#      run: Invoke-WebRequest -Uri "https://www2.graphviz.org/Packages/stable/windows/10/cmake/Release/x64/graphviz-install-2.44.1-win64.exe" -OutFile "C:\Temp\graphviz-install-2.44.1-win64.exe"
-#
-#    - name: Install exe
-#      run: Start-Process -Wait -FilePath "C:\Temp\graphviz-install-2.44.1-win64.exe" -ArgumentList '/S' -PassThru
-#
-#    - name: Install packages
-#      run: |
-#        $PATH = [Environment]::GetEnvironmentVariable("PATH")
-#        $graphviz_path = "C:\Program Files\Graphviz 2.44.1\bin"
-#        [Environment]::SetEnvironmentVariable("PATH", "$graphviz_path;$PATH")
-#        dot -c
-#        dot -v
-#        pip install --upgrade pip wheel setuptools --user
-#        pip install -r requirements/test.txt --user
-#        python setup.py build_ext --include-dirs="C:\Program Files\Graphviz 2.44.1\include" --library-dirs="C:\Program Files\Graphviz 2.44.1\lib"
-#        python setup.py install --user
-#        pip list
-#
-#    - name: Test pygraphviz
-#      run: |
-#        $PATH = [Environment]::GetEnvironmentVariable("PATH")
-#        $graphviz_path = "C:\Program Files\Graphviz 2.44.1\bin"
-#        [Environment]::SetEnvironmentVariable("PATH", "$graphviz_path;$PATH")
-#        [Environment]::SetEnvironmentVariable("PY_IGNORE_IMPORTMISMATCH", 1)
-#        pytest --doctest-modules --durations=10 --pyargs pygraphviz
+        $PATH = [Environment]::GetEnvironmentVariable("PATH")
+        $graphviz_path = "C:\Program Files\Graphviz\bin"
+        [Environment]::SetEnvironmentVariable("PATH", "$graphviz_path;$PATH")
+        [Environment]::SetEnvironmentVariable("PY_IGNORE_IMPORTMISMATCH", 1)
```

Comment:
You may want to remove this.  I put it in b/c there was some import mismatch issue.  I can't remember what the issue was.  But maybe this is hidding what the problem is.  And I just got lucky that I was able to ignore it on Python 3.7.